configs/typescript.example.ts

```diff
@@ -1,5 +1,6 @@
 // Use --report-unused-disable-directives to validate errors
```

Comment:
Renamed this file to `.example.ts` instead of `.test.ts` so it doesn't get run under jest, only linted.